decentraland/unity-renderer
# Profanity avatar names filtering

## What does this PR change?

Adds profanity filtering to user names. So if a user's name contains an offensive word it should be filtered with ****:

- General chat
- Avatar name tags
- Player info card (description is also filtered)

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/profanity-avatar-names-filtering
2. Login with a friend who has an offensive name like "shithead". Also set a description with offensive words at the profile menu.
3. Write anything on the general chat: your friend's name should be filtered ****head
5. The avatar name tag should be filtered
6. Open the profile info card by clicking on your friend's avatar. The name and description should be filtered

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
